### PR TITLE
Slight Rimac Buff

### PR DESCRIPTION
**Pull request by cxerq**

### CODE REVIEW

File: DATA/rimac_data/handling.meta [hunk 1]
```diff
@@ -14,7 +14,7 @@
 			<fDriveInertia value="1.000000"/>
 			<fClutchChangeRateScaleUpShift value="9.000000"/>
 			<fClutchChangeRateScaleDownShift value="9.000000"/>
-			<fInitialDriveMaxFlatVel value="243.900000"/>
```
Comment (fkk-cz) [inline DATA/rimac_data/handling.meta:17]: This old value implies a top speed of ~199.26 MPH. If the vehicle is 'only' doing 180 that indicates `fInitialDragCoeff` is the issue. Try lowering the value to around 5.0 to see if that helps.

**Review by fkk-cz — CHANGES_REQUESTED** [review]

**Comment by fkk-cz** [comment]
Closes #1018.

**Comment by Stagonium** [comment]
This brought the top speed I could reach to 181mph, up from 175mph.

**Comment by glefave0209** [comment]
I got the same result. It governs out at 182

**Comment by No-Yes1** [comment]
Make another vehicle issue.

**Pull request merged**